packages/backend/src/controllers/issues/issues.controller.ts

```diff
@@ -132,6 +133,37 @@ export class IssueController {
     );
   }
 
+  @Put(':id/resolve')
+  @ApiOperation({ summary: 'Mark a user as having reesolved an issue.' })
+  @ApiOkResponse({
+    description: 'Issue resolved successfully',
+    type: IssueResponseDto,
+  })
+  async updateBillPayment(
```

Comment:
can you change the name of `updateBillPayment` to `updateIssueResolved`  @sher812 and any other Bill/Issue discrepancies pls?

Reply:
oh whoops

Reply:
will do, do you also mean the variable names in the issue details page?

Reply:
yea, if you feel like it. not sure if anyone else has been assigned to do it yet. but it really helps my brain be less confused when reading these issues files 😅